tailwind.config.js

```diff
@@ -4,6 +4,8 @@ const periodicTableOfRealtimeColors = {
   'low-level-transport': 'var(--color-jazzy-pink)',
   'application-transport': '#e40060',
   'realtime-transport': '#ff2739',
+  'api-reference-attribute-highlight': '#ffe6dc',
+  'api-reference-attribute-border': '#ff9e7a',
```

Comment:
This should be added in a seperate hash, these colors are very specific to the periodic realtime table ... which is in voltaire, not here ??? :)

Reply:
Some of those colours are (or maybe only were) used a lot elsewhere in Voltaire, so when I was taking components from Voltaire early in docs development I thought this was a metaphorical periodic table of colours to be used to style anything; happy to make a separate hash for this.

Reply:
yes please, most of these are one of colors for https://ably.com/periodic-table-of-realtime. You can just rename the hash the leave the colors you want.